acceptance_tests/features/pages/add_survey.py

```diff
@@ -0,0 +1,11 @@
+from acceptance_tests import browser
+from config import Config
+
+
+def go_to():
```

Comment:
Should we update `go_to` on each of these to be `go_to_add_survey` or something similar?

Reply:
I agree this can be addressed in a different PR

Reply:
At the moment we just followed what has already been done. It seems fine currently as when you call the go_to the page is also there so for readability it seems fine just now. But could be changed in the future.